Created Dictionary Pair Programming Post

intjnotinfp opened the pull request:
Created first draft of Kevin and Audrey's dictionary exercises pair programming reflections. Will add reflection after it is successfully set up as a co-authored post!

Commit:
Created Dictionary Pair Programming Post

Created first draft of Kevin and Audrey's dictionary exercises pair programming reflections. Will add reflection after it is successfully set up as a co-authored post!

Commit:
Added reflection to dict pair exercise post

Added reflection to the code that was already there. Also switching from 'draft' to requesting review. Please review/merge or request any necessary changes!

MishaCienfuegos commented on _posts/intjnotinfp/2020-10-20-dictionary-pair-exercise-kevin-audrey:
```diff
@@ -0,0 +1,28 @@
+---
+author:
```
Looks like this is missing the `layout: post`.

KJEZ35 replied:
@MishaCienfuegos went back and fixed this, thank you!

KJEZ35 replied:
Thank you for the catch @MishaCienfuegos, went back and added this.

MishaCienfuegos reviewed (changes requested):
This is looking good! It does appear the file is missing the `.md`. Could you update that?

KJEZ35 committed:
Updated_Paired_Program 

Went back to add the ".md" to our post.

KJEZ35 reviewed (approved):
Went back to add the .md

eah13 commented:
@KJEZ35 Nice work on the file extension. 

The post is still missing a `layout: post` in the YAML header (see comment from Misha above). Your comment there says you addressed this, but I don't see that in the code. One more quick fix and this is ready for merge

KJEZ35 committed:
Update_to_paired_programming

Forgot to add the layout post, in order to merge the pull request into production.

KJEZ35 reviewed (approved):
Added the layout, 11-06-2020

MishaCienfuegos reviewed (approved):
Awesome, looks good! 🍂
